src/assets/src/validation.ts

```diff
@@ -66,7 +66,8 @@ export const queueLocationSchema = string().defined().trim().max(100, createRema
 export const uniqnameSchema = string().defined().trim().lowercase()
     .min(3, 'Uniqnames must be at least 3 characters long.')
     .max(8, 'Uniqnames must be at most 8 characters long.')
-    .matches(/^[a-z]+$/i, 'Uniqnames cannot contain non-alphabetical characters.');
+    .matches(/^[^@]*$/, "Please use your uniqname instead of your email address.")
```

Comment:
Looks like a better error message should be "Please use uniqname instead of email address."

Reply:
Message has been changed as well. 
![image](https://github.com/tl-its-umich-edu/remote-office-hours-queue/assets/56746786/db61aab2-fcbd-49ee-b52f-f634ffa4a08e)